BenchmarkVCFs/vcfdistBenchmark.wdl

```diff
@@ -0,0 +1,71 @@
+
+version 1.0
+
+workflow vcfdist_eval {
```

Comment:
Just a stylistic comment: I'd make workflow/task names using CapitalCamelCase, e.g. `VcfdistEval`, etc. Same for the `.wdl` filename.